pkg/types/tool.go

```diff
@@ -780,3 +785,7 @@ func FirstSet[T comparable](in ...T) (result T) {
 	}
 	return
 }
+
+func isAlphaNumeric(s string) bool {
+	return regexp.MustCompile(`^[a-zA-Z0-9_]+$`).MatchString(s)
```

Comment:
does this also need to include `.` to support common aliases in use today? e.g. `sys.openai` https://github.com/gptscript-ai/dalle-image-generation/blob/763a867b4e8ae0529782b74c1bcb08ecb197b2c8/tool.gpt#L2

Reply:
Thanks...I am very dumb for not thinking of this, lol

Reply:
heh, the only reason I thought of it is because I was just looking at it lol